### PR TITLE
[13.0][MIG] sale_order_product_recommendation: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT21669

### CODE REVIEW

**Comment by Tardo** [comment]
Idk why runbot is red :/

**Comment by Tardo** [comment]
Ok, all green! ready to review

File: sale_order_product_recommendation/wizards/sale_order_recommendation.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            so_line_ids.append(so_line.id)
+        self.order_id.order_line += so_line_obj.browse(so_line_ids)
+
+
+class SaleOrderRecommendationLine(models.TransientModel):
+    _name = "sale.order.recommendation.line"
+    _description = "Recommended product for current sale order"
+    _order = "id"
+
+    currency_id = fields.Many2one(related="product_id.currency_id", readonly=True)
```
Comment (sergio-teruel) [inline sale_order_product_recommendation/wizards/sale_order_recommendation.py]: Remove readonly=True, by default is True

**Review by sergio-teruel — APPROVED** [review]
Only a little comment

**Comment by Tardo** [comment]
Done!

**Comment by Tardo** [comment]
This PR need reviews

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@pedrobaeza Following the conversation on contributors mailing list, don't use nobump as it won't generate wheels to pypi. Use patch instead (even if it will generate a 13.0.1.0.1 version)

**Comment by pedrobaeza** [comment]
Sorry, but I don't like that. And any way you have to deploy with --pre for pip for being sure, so it doesn't add anything.

**Comment by rousseldenis** [comment]
> Sorry, but I don't like that. And any way you have to deploy with --pre for pip for being sure, so it doesn't add anything.

What don't you like ?

So when using nombump, the module on pypi won't be marked as release one :

image

And the --pre option is usually used for development (even if for releases you should use the version directly).

**Comment by pedrobaeza** [comment]
I don't like to have a recently migrated module with version 13.0.1.0.1 and people wondering where it's the previous version. And you know that I don't think pypi is the ideal way of distributing this, but let's not starting again the same debate.

**Pull request merged**